### ISSUE TITLE
Dynamic island feature 

### ISSUE THREAD

**Issue by saran8796** [25huizengek1/ViTune#617]
Please include an music playback live alert(dynamic island) in OnePlus mobile like Spotify, include some useful functions in the live alert.

**Comment by dtomvan** [comment]
What would that look like? I'm not familiar with the dynamic island feature on oneplus. Could you maybe provide a screenshot/screencast of what that looks like?

**Comment by 25huizengek1** [comment]
@saran8796 OxygenOS 15 is just rolling out with this new feature (Live alerts), and there seems to be no official way for individual apps to support this. People on forums claim it works for VLC media player, so I'll see if that leads to anything...

**Comment by 25huizengek1** [comment]
I cannot find anything that leads to something like this. OnePlus seems to have documented almost nothing themselves, and I don't own any OnePlus devices so I'm afraid I cannot build this myself.

**Comment by saran8796** [comment]
> What would that look like? I'm not familiar with the dynamic island feature on oneplus. Could you maybe provide a screenshot/screencast of what that looks like?

![Screenshot_2024-12-16-06-04-44-87_b783bf344239542886fee7b48fa4b892](https://github.com/user-attachments/assets/a1db3a0f-7401-4771-b5c2-635c8da5ccf8)
![Screenshot_2024-12-16-06-05-28-84_b783bf344239542886fee7b48fa4b892](https://github.com/user-attachments/assets/b586abec-9af2-44f6-8561-742b45696b5d)

**Comment by saran8796** [comment]
> > What would that look like? I'm not familiar with the dynamic island feature on oneplus. Could you maybe provide a screenshot/screencast of what that looks like?
> 
> ![Screenshot_2024-12-16-06-04-44-87_b783bf344239542886fee7b48fa4b892](https://private-user-images.githubusercontent.com/67047833/395923703-a1db3a0f-7401-4771-b5c2-635c8da5ccf8.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzMDk2ODUsIm5iZiI6MTczNDMwOTM4NSwicGF0aCI6Ii82NzA0NzgzMy8zOTU5MjM3MDMtYTFkYjNhMGYtNzQwMS00NzcxLWI1YzItNjM1YzhkYTVjY2Y4LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDAwMzYyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJmNDBhMGMyYzg5NTZhZTY1NWU4N2ZhYTMxNWVlODYyZWI0ZTJkMDRjZTE1ZTI2ODlmOTcyNWJjOTRjMTJlMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mi-E_ed0ImE9qX4PI3Dyh-j6_7iX-YbVtA3benGmEGI) ![Screenshot_2024-12-16-06-05-28-84_b783bf344239542886fee7b48fa4b892](https://private-user-images.githubusercontent.com/67047833/395923710-b586abec-9af2-44f6-8561-742b45696b5d.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzMDk2ODUsIm5iZiI6MTczNDMwOTM4NSwicGF0aCI6Ii82NzA0NzgzMy8zOTU5MjM3MTAtYjU4NmFiZWMtOWFmMi00NGY2LTg1NjEtNzQyYjQ1Njk2YjVkLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDAwMzYyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyOTA1YTlhZjk4OWMyYzE0YTA3MzBhY2EzMTA2ZGUxZjMyZDlkNGQ1NjIwZDNkOGNmMGY0MmM2MDExODY1ZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hpQMk-Tnwszw-yrQNfps4btjOc2yHi4z5dF2xwdCgeA)


This is the live alerts oneplus launched in oxygen os 15 this support vlc, Spotify so i asked to make it supported in all devices which supports dynamic island features not only in OnePlus. Thank you

**Comment by Sparx2OP** [comment]
For now, you can use any APK editor app and change the package name to "com.spotify.music". Everything works fine including play pause in Live alerts. Or if you're rooted you can use a module (OPlus FeatureMusicMediaFluid) to get that live alert without modifying apk.

**Comment by dtomvan** [comment]
Wait, so you mean the support for such a dynamic island does not sprout from anything Spotify does, but just the fact that OnePlus added it to their "Yes, these apps are music players and others aren't" list?

If so, I'd have to close this as it isn't something we can do anything about. The only thing I could tell you is that you could maybe reach out to OnePlus on some kind of feedback form they probably have, and pray they add it.

**Comment by Sparx2OP** [comment]
> Wait, so you mean the support for such a dynamic island does not sprout from anything Spotify does, but just the fact that OnePlus added it to their "Yes, these apps are music players and others aren't" list?
> 
> If so, I'd have to close this as it isn't something we can do anything about. The only thing I could tell you is that you could maybe reach out to OnePlus on some kind of feedback form they probably have, and pray they add it.

Yep, that's true, Only OnePlus can add live alerts to specific apps and it doesn't matter if that app needs support for live alerts. (I've tried the change package name method and again, everything works fine in Live alerts)

**Comment by dtomvan** [comment]
Alright, seems like there's nothing the project can do to address this. Closing.
